modules/age.nix

```diff
@@ -90,6 +90,17 @@ let
           Group of the decrypted secret.
         '';
       };
+      action = mkOption {
+        type = types.str;
+        default = "";
+        description = "A script to run when secret is updated.";
+      };
+      services = mkOption {
+        type = types.listOf types.str;
+        default = [];
+        description = "The systemd services that uses this secret. Will be restarted when the secret changes.";
+        example = "[ wireguard-wg0 ]";
+      };
```

Comment:
I'm glad you weighed in. I don't have strong opinions about any of this, and the only reason I didn't do it the way you're describing is because I'm not that familiar with systemd units. Feel free to make these changes.

Reply:
Oh. You don't have access to my repository.* I guess make a PR against my fork and I'll accept it? I would just make the changes myself but I don't think I understand them well enough to do it right on my first try.

*I stop using git for a week and totally forget how anything works XD